Add Insert citations

crsh opened the pull request:
Hi, thanks for this great list. I'd like to add my new addin to insert markdown citations.

daattali commented:
Very cool. But could you add it at the bottom of the readme instead of a random location in the middle?

crsh commented:
I added it to that location because I thought it went well with the other R Markdown related addins but I have now moved it to the bottom of the list.

daattali commented:
I didn't notice that there were a few rmarkdowns in order, that was a coincidence :)
